tools/clean-filestore-limit.sh

```diff
@@ -13,8 +13,17 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# shellcheck disable=SC2317
+# known issue that shellcheck wrongly detects trap function as unreachable
+
 set -e -o pipefail
 
+function enable_filestore_api() {
+	echo "Re-enabling Filestore API..."
+	gcloud services enable file.googleapis.com --project "${PROJECT_ID}"
+	exit 0
```

Comment:
won't this eat up any error that happens in the script? 
Say deleting a filestore instance failed, then it calls the trap, but then the trap has an exit 0... 
What do you think?

Reply:
Saw that too. Addressed.